packages/fluentui/docs/package.json

```diff
@@ -67,11 +67,13 @@
   },
   "scripts": {
     "build": "gulp build:docs",
-    "build:storybook": "build-storybook -o dist/storybook",
+    "prebuild:storybook": "lage build:info --to @fluentui/react-northstar @fluentui/react-component-ref @fluentui/react-bindings",
```

Comment:
**Why is this needed?**

stories that are being build for VR tests contain paths that use  following webpack constructs

```
// @filename packages/fluentui/docs/src/utils/componentInfoContext.ts

const infoObjects = importAll([
  require.context('@fluentui/react-component-ref/componentInfo', true, /\.info\.json$/),
  require.context('@fluentui/react-bindings/componentInfo', true, /\.info\.json$/),
  require.context('@fluentui/react-northstar/componentInfo', true, /\.info\.json$/),
]);
```

This forces storybook to parse this , but without those being present webpack will explode. That's the reason we need to build these artifacts prior to running webpack - although they are not used at all for VR. 

While far from ideal this dramatically improves storybook start/build speeds on local/CI 🙌